TriangleApp/app/src/main/java/triangle/triangleapp/presentation/impl/StreamActivity.java

```diff
@@ -28,7 +28,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_camera);
 
-        mPresenter = new StreamPresenter(this);
+        mPresenter = new StreamPresenter(this, getApplicationContext());
```

Comment:
We dont not want the manager or presenter to have access to the Context. 
Better to say we dont want to pass context around (Major cause of memory leaks)